app/src/main/java/justforcommunity/radiocom/activities/FirebaseActivity.java

```diff
@@ -1,8 +1,8 @@
 /*
  *
- *  * Copyright (C) 2016 @ Fernando Souto González
+ *  * Copyright (C) 2017 @ Pablo Grela
```

Comment:
I remove ResponseProgramDTO, with RadioCo is not use.
Maybe create branch CUACFM of radiocom-android